common/lib/models/reply.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:opengov_common/common.dart';
+
+part 'reply.g.dart';
+
+abstract class ReplyBase {
```

Comment:
Are there any other classes that inherit from ReplyBase? If not, you can get rid of it and just use Reply directly. I made `CommentBase` as a superclass for `Comment` and `FeedComment`, but I don't think we'll need something similar for replies.